cloud.ganttproject.colloboque/src/main/kotlin/cloud/ganttproject/colloboque/DevServer.kt

```diff
@@ -108,14 +108,14 @@ class ColloboqueWebSocketServer(port: Int, private val colloboqueServer: Collobo
         )
         send(Json.encodeToString(response))
       } catch (e: Exception) {
-        LOG.error("Failed to send response", e)
+        LOG.error("Failed to send response {}", e)
       }
     }
 
     override fun onPong(pong: WebSocketFrame?) {}
 
     override fun onException(exception: IOException?) {
-      LOG.error("WebSocket exception $exception")
+      LOG.error("WebSocket exception {}", exception = exception)
```

Comment:
```suggestion
      LOG.error("WebSocket exception {}", exception)
```

Reply:
It's not happy cause the value is nullable, it wants it to be Any in this case

Reply:
I think that it is nullable just because  the code generator could not prove the opposite when it was implementing this method. IRL it makes no sense to call onException method with null argument.

Anyway, `exception = exception` is ok too (and may be even better) but remove the placeholder then, as there is no value to put into the placeholder. Exception argument is handled separately.